### ISSUE TITLE
Property `collapsable` => `collapsible` for components

### ISSUE THREAD

**Issue by AlexKVal** [react-bootstrap/react-bootstrap#589]
Rename property `collapsable` => `collapsible` for
- [ ] Nav
- [ ] Panel
- [ ] PanelGroup

Additional task after #425

**Comment by mtscout6** [comment]
Why the need for a separate issue on this, why not do this with #425?

**Comment by AlexKVal** [comment]
I thought about these little and nice looking progress bars :smile: 

(honestly: I was just so engrossed in code moving between a number of branches and commits,
those `git add -p / git commit --fixup / git rebase -i --autosquash / git cherry-pick`,
so I hadn't thought through all the picture)

I also prefer do not spawn additional issues if it's feasible.
Closed and moved to 425.
